src/tasks.js

```diff
@@ -0,0 +1,92 @@
+export default class Tasker {
+  constructor() {
+    this.tasksArray = JSON.parse(localStorage.getItem('tasks')) || [];
+    this.populateTodo();
+  }
+
+  populateTodo() {
+    this.save();
+    const listSec = document.querySelector('#list-items');
+    listSec.innerHTML = '';
+    this.tasksArray.forEach((taskItem) => {
+      let activityItem = `
+        <li class="d-flex s-between list-item" id="item-data-${taskItem.index}">`;
+      activityItem += `
+            <span class="material-icons  update-status"  data-id="${taskItem.index}">
+              check_box_outline_blank
+            </span>
+            <p contenteditable="true" class="activity" data-id="${taskItem.index}">
+              ${taskItem.description}
+            </p>`;
+
+      activityItem += `
+          <span class="material-icons delete-activity"  data-id="${taskItem.index}">
+            delete
+          </span>
+        </li>
+      `;
+      listSec.innerHTML += activityItem;
+    });
+    this.Actions();
+  }
+
+  // Add activity
+  add(actv) {
+    if (actv || actv === 0) {
+      const newActivity = {
+        description: actv,
+        completed: false,
+        index: this.tasksArray.length,
+      };
+      this.tasksArray.push(newActivity);
+      this.populateTodo();
+    }
+  }
```

Comment:
- [ ] When you update the task as completed to be true, the changes are not reflected or stored in the local storage, or if you reload the page you can't any changes.

![image](https://user-images.githubusercontent.com/55635977/165974391-9e9a44a2-6db3-4d08-9c6a-7bae29886fd3.png)

Reply:
@mwenyoa Kindly check the comment now.

Reply:
You need to have the functionality of the checked item in this milestone. Even though in this milestone I need to review your project based on the `add` and `remove` functionalities as per project requirements. Your checked items functionality needs to be working as well because it was required in the "Interactive list" milestone before this one.

Kindly apply the functionality of the checked item as requested.